#Header header

PointedObject[] pointed_objects
